moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
         },
-        "prev": {
-          "type": "string",
-          "format": "iri",
-          "description": "The previous page if available"
-        }
-      }
-    },
-    "data": {
-      "type": "array",
-      "description": "The actual data of the response",
-      "uniqueItems": true,
-      "items": {
-        "title": "Course",
-        "type": "object",
-        "description": "Serialized course object",
-        "required": [
-          "id",
-          "type",
-          "attributes"
-        ],
-        "properties": {
-          "id": {
+        "inLanguage": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "uniqueItems": true,
+          "description": "List of languages in which the course is offered. Each language should be abbreviated in accordance with BCP 47: https://tools.ietf.org/html/bcp47.",
+          "items": {
             "type": "string",
-            "minLength": 1,
-            "description": "ID of the object, UUID is preferred"
-          },
-          "type": {
+            "example": "en"
+          }
+        },
+        "startDate": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "uniqueItems": true,
+          "description": "The time and date the course starts. It is possible to give several date-times for repetitive courses. The date-time to be given according to ISO 8601.",
+          "items": {
             "type": "string",
-            "enum": [
-              "courses"
-            ],
-            "description": "Type of the object"
+            "format": "date-time",
+            "example": "2021-01-13T08:00:00Z"
+          }
+        },
+        "endDate": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "uniqueItems": true,
+          "description": "The time and date the course ends. It is possible to give several date-times for repetitive courses. The date-time to be given according to ISO 8601.",
+          "items": {
+            "type": "string",
+            "format": "date-time",
+            "example": "2021-01-27T20:00:00Z"
+          }
+        },
+        "expires": {
```

Comment:
We have active verb and noun wording mixed up. startDate and endDate but expires (vs. expiryDate or similar.)

Reply:
This is based on schema.org vocabulary.